sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/object.scala

```diff
@@ -286,6 +290,9 @@ case class FlatMapGroupsInR(
     child: LogicalPlan) extends UnaryNode with ObjectProducer{
 
   override lazy val schema = outputSchema
+
+  override protected def stringArgs: Iterator[Any] = Iterator(inputSchema, outputSchema,
+    outputObjAttr, child)
```

Comment:
Ah I see - Sorry. I think I saw the diff from one commit there. Also it looks like that commit reverted the override change to the MapPartitionsInR class ?  I think thats useful to have. Do you want to do  that in a separate PR ?